README.md

```diff
@@ -33,11 +33,19 @@ Developers on a OS for which there is no binary package, or who would like to st
 1. Install the Xcode command-line tools.
 1. Download version 1.52 of [Boost](https://sourceforge.net/projects/boost/files/boost/1.52.0/).
 1. Set the `BOOSTDIR` environment variable to the location containing this boost installation.
-1. Install [Mono](http://www.mono-project.com/download/stable/).
-1. Install a [JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html). FoundationDB currently builds with Java 8.
+1. Install [Mono] from ports(https://www.freshports.org/lang/mono).
+1. Install a [JDK] from ports(http://www.oracle.com/technetwork/java/javase/downloads/index.html). FoundationDB currently builds with Java 8.
 1. Navigate to the directory where you checked out the foundationdb repo.
 1. Run `make`.
 
+#### FreeBSD
+
+1. Check out this repo on your server.
+1. Install dependencies from ports using the following command: pkg install mono boost-libs gmake bash
+1. (Optional) Install a [JDK](https://www.freshports.org/java/openjdk8/) for Java Bindings. FoundationDB currently builds with Java 8.
+1. Navigate to the directory where you checked out the foundationdb repo.
+1. Install [gmake](https://www.freshports.org/devel/gmake/) from ports and Run `gmake`.
+
```

Comment:
To compile the go bindings, one also needs to `pkg install lang/go`.  It appears that go bindings do not compile.

Reply:
ok, maybe this will give me a chance to learn a little about go as well. For the testing I've done so far I started with Python which I feel pretty comfortable with and used that to test the symbols and work from there. I did not try just building with 'gmake bindings' so aside from Python and NodeJS those were put in as sort of placeholders until those could be worked on. If other bindings are actually building that's great!

Reply:
You are correct, "Go" is needed for this and seems to build now. I need to test that something working is being built now but it is creating the bins for "Go" successfully